.buildkite/_forge.rayci.yml

```diff
@@ -126,3 +129,13 @@ steps:
   - name: mlbuild
     wanda: ci/docker/ml.build.wanda.yaml
     depends_on: oss-ci-base_ml
+    env:
+      FROM: oss-ci-base_ml
+      DESTINATION: mlbuild
+
+  - name: mlgpubuild
+    wanda: ci/docker/ml.build.wanda.yaml
+    depends_on: oss-ci-base_gpu
+    env:
+      FROM: oss-ci-base_gpu
+      DESTINATION: mlgpubuild
```

Comment:
could you make these two env var's longer? maybe just `BASE_BUILD`? or `IMAGE_FROM`, `IMAGE_TO`? or `IMATE_SRC`, `IMAGE_DEST`.

it is often from/to , source/destitnation. combining from and destination is a bit.. unconventional.

Reply:
Ahh, I also feel it unnatural but did not know why ;)